src/securemem.rs

```diff
@@ -1,9 +1,5 @@
+use zeroize::Zeroize;
+
 pub fn zero(to_zero: &mut [u8]) {
-    // the unsafety of this call is bounded to the existence of the pointer
-    // and the accuracy of the length of the array.
-    //
-    // since to_zero existence is bound to live at least as long as the call
-    // of this function and that we use the length (in bytes) of the given
-    // slice, this call is safe.
-    unsafe { ::std::ptr::write_bytes(to_zero.as_mut_ptr(), 0, to_zero.len()) }
+    to_zero.zeroize()
```

Comment:
Why not simply using the `core`'s function for `write_bytes` instead of adding a new dependency ?

```suggestion
unsafe { ::core::ptr::write_bytes(to_zero.as_mut_ptr(), 0, to_zero.len()) };
```

documentation on [`write_bytes`](https://doc.rust-lang.org/core/ptr/fn.write_bytes.html)

Reply:
Yes, you are right. Just modified to use this method. 😄